tests/lib/rules/html-indent.js

```diff
@@ -381,6 +381,33 @@ tester.run('html-indent', rule, loadPatterns(
         text <span /> <!-- comment --></pre>
         </template>
       `
+    },
+    // Close bracket
+    {
+      filename: 'test.vue',
+      code: unIndent`
+        <template>
```

Comment:
If the source code where all lines are indented, You can be moved to fixtures.

Reply:
I moved these examples to fixtures. And turns out there is one more issue.

The following code is fine:
```html
<!--{"options":[2, { "closeBracket": 1 }]}-->
<template>
  <a
    href="#"
    >Link 1</a
  >
</template>
```

But when there is any element below the link, like here:
```html
<!--{"options":[2, { "closeBracket": 1 }]}-->
<template>
  <a
    href="#"
    >Link 1</a
  >
  <span>text</span>
</template>
```
it wants to additionally indent the close bracket of </a> like this:
```html
<template>
  <a
    href="#"
    >Link 1</a
    >
  <span>text</span>
</template>
```

Do you have an idea what might be causing it @mysticatea ?